e2e/cypress/integration/integrations/integrations_spec.js

```diff
@@ -178,4 +182,25 @@ describe('Integrations page', () => {
         // * Validate that the correct empty message is shown
         cy.get('#emptySearchResultsMessage').should('be.visible').and('have.text', `No bot accounts match ${searchString}`);
     });
+
+    it('MM-T570 Integration Page titles are bolded', () => {
+        cy.visit(`/${testTeam.name}/channels/town-square`);
+
+        // # Go to Main Menu -> Integrations
+        cy.get('#sidebarHeaderDropdownButton').should('be.visible').click();
+
+        cy.get('#integrations').should('be.visible').click();
+
+        cy.get('.integration-option__title').contains('Incoming Webhooks').click();
+
+        integrationPageTitleIsBold('Incoming Webhooks');
+        integrationPageTitleIsBold('Outgoing Webhooks');
+        integrationPageTitleIsBold('Slash Commands');
+        integrationPageTitleIsBold('OAuth 2.0 Applications');
```

Comment:
I recently updated the test to include check for Bot accounts. Could please insert,
```javascript
integrationPageTitleIsBold('Bot Accounts');
```